consumer.go

```diff
@@ -299,6 +299,9 @@ type partitionConsumer struct {
 	errors   chan *ConsumerError
 	feeder   chan *FetchResponse
 
+	replicaInited        bool
```

Comment:
initiated? what is this going to be used for?

Reply:
I will rename it...its used to fetch from read replica instead of master.